webview/src/experiments/components/AddStage.tsx

```diff
@@ -5,16 +5,28 @@ import { IconButton } from '../../shared/components/button/IconButton'
 import { Add } from '../../shared/components/icons'
 import { sendMessage } from '../../shared/vscode'
 
-export const AddStage: React.FC = () => (
-  <div className={styles.addConfigButton}>
+interface AddStageProps {
+  hasValidDvcYaml: boolean
+}
+
+export const AddStage: React.FC<AddStageProps> = ({ hasValidDvcYaml }) => (
+  <div className={styles.addConfigButton} data-testid="aaa">
```

Comment:
Should we name this id to be a more clear name? Though it looks like it isn't being used anywhere 🤔

Reply:
Nice catch It was for quick testing (had a lot of trouble testing the shadow DOM button), but I forgot to remove it after.